client/src/components/organisms/Creator/Creator.jsx

```diff
@@ -50,19 +54,20 @@ function Creator({kind}) {
     }
     
     return (
-        <div>
+        (creatorDetails !== null && creatorVideos) ?
+        (<div>
         {
-            (creatorDetails !== "Loading...") ? <CreatorDetails name={creatorDetails.name} description={creatorDetails.description} thumbnail={creatorDetails.thumbnail} handleFollow={handleFollow} isFollowing={(currentUserId !== null) ? searchFollowings(videoState,creatorDetails._id): false} currentUserId={currentUserId}/>: creatorDetails
+            (creatorDetails !== null) ? <CreatorDetails name={creatorDetails.name} description={creatorDetails.description} thumbnail={creatorDetails.thumbnail} handleFollow={handleFollow} isFollowing={(currentUserId !== null) ? searchFollowings(videoState,creatorDetails._id): false} currentUserId={currentUserId}/>: creatorDetails
```

Comment:
Give proper indentation as double ternary operator is making it look confusing

Reply:
Thanks. Have updated :)